bedrock/releasenotes/tests/data/firefox_history_major_releases.json

```diff
@@ -1,42 +1 @@
-{
-    "1.0": "2004-11-09",
-    "1.5": "2005-11-29",
-    "2.0": "2006-10-24",
-    "3.0": "2008-06-17",
-    "3.5": "2009-06-30",
-    "3.6": "2010-01-21",
-    "4.0": "2011-03-22",
-    "5.0": "2011-06-21",
-    "6.0": "2011-08-16",
-    "7.0": "2011-09-27",
-    "8.0": "2011-11-08",
-    "9.0": "2011-12-20",
-    "10.0": "2012-01-31",
-    "11.0": "2012-03-13",
-    "12.0": "2012-04-24",
-    "13.0": "2012-06-05",
-    "14.0.1": "2012-07-17",
-    "15.0": "2012-08-28",
-    "16.0": "2012-10-09",
-    "17.0": "2012-11-20",
-    "18.0": "2013-01-08",
-    "19.0": "2013-02-19",
-    "20.0": "2013-04-02",
-    "21.0": "2013-05-14",
-    "22.0": "2013-06-25",
-    "23.0": "2013-08-06",
-    "24.0": "2013-09-17",
-    "25.0": "2013-10-29",
-    "26.0": "2013-12-10",
-    "27.0": "2014-02-04",
-    "28.0": "2014-03-18",
-    "29.0": "2014-04-29",
-    "30.0": "2014-06-10",
-    "31.0": "2014-07-22",
-    "32.0": "2014-09-02",
-    "33.0": "2014-10-14",
-    "33.1": "2014-11-10",
-    "34.0": "2014-12-01",
-    "35.0": "2015-01-13",
-    "36.0": "2015-02-24"
-}
+{"1.0": "2004-11-09", "1.5": "2005-11-29", "2.0": "2006-10-24", "3.0": "2008-06-17", "3.5": "2009-06-30", "3.6": "2010-01-21", "4.0": "2011-03-22", "5.0": "2011-06-21", "6.0": "2011-08-16", "7.0": "2011-09-27", "8.0": "2011-11-08", "9.0": "2011-12-20", "10.0": "2012-01-31", "11.0": "2012-03-13", "12.0": "2012-04-24", "13.0": "2012-06-05", "14.0.1": "2012-07-17", "15.0": "2012-08-28", "16.0": "2012-10-09", "17.0": "2012-11-20", "18.0": "2013-01-08", "19.0": "2013-02-19", "20.0": "2013-04-02", "21.0": "2013-05-14", "22.0": "2013-06-25", "23.0": "2013-08-06", "24.0": "2013-09-17", "25.0": "2013-10-29", "26.0": "2013-12-10", "27.0": "2014-02-04", "28.0": "2014-03-18", "29.0": "2014-04-29", "30.0": "2014-06-10", "31.0": "2014-07-22", "32.0": "2014-09-02", "33.0": "2014-10-14", "33.1": "2014-11-10", "34.0": "2014-12-01", "35.0": "2015-01-13", "36.0": "2015-02-24", "37.0": "2015-03-31", "38.0": "2015-05-12", "39.0": "2015-07-02", "40.0": "2015-08-11", "41.0": "2015-09-22", "42.0": "2015-11-03", "43.0": "2015-12-15", "44.0": "2016-01-26", "45.0": "2016-03-08", "46.0": "2016-04-26", "47.0": "2016-06-07", "48.0": "2016-08-02", "49.0": "2016-09-20", "50.0": "2016-11-15"}
```

Comment:
A nit-picky request, but if you have time, could you pretty print the new JSON? Just makes it easier to scan.